### ISSUE TITLE
Using taggit together with simple history makes migrations non-portable

### ISSUE THREAD

**Issue by amcpherson** [treyhunner/django-simple-history#317]
When I use the following to track history of tags:

```
import taggit.models
import simple_history
simple_history.register(taggit.models.Tag)
```

makemigrations creates migrations in the taggit migrations directory under site packages.  If i then try to bring up the database on another server, those migrations are not found:

```
Traceback (most recent call last):
  File "/Users/amcphers/Anaconda/envs/tantalus/lib/python2.7/site-packages/django/utils/autoreload.py", line 228, in wrapper
    fn(*args, **kwargs)
  File "/Users/amcphers/Anaconda/envs/tantalus/lib/python2.7/site-packages/django/core/management/commands/runserver.py", line 128, in inner_run
    self.check_migrations()
  File "/Users/amcphers/Anaconda/envs/tantalus/lib/python2.7/site-packages/django/core/management/base.py", line 422, in check_migrations
    executor = MigrationExecutor(connections[DEFAULT_DB_ALIAS])
  File "/Users/amcphers/Anaconda/envs/tantalus/lib/python2.7/site-packages/django/db/migrations/executor.py", line 20, in __init__
    self.loader = MigrationLoader(self.connection)
  File "/Users/amcphers/Anaconda/envs/tantalus/lib/python2.7/site-packages/django/db/migrations/loader.py", line 52, in __init__
    self.build_graph()
  File "/Users/amcphers/Anaconda/envs/tantalus/lib/python2.7/site-packages/django/db/migrations/loader.py", line 274, in build_graph
    raise exc
django.db.migrations.exceptions.NodeNotFoundError: Migration tantalus.0001_initial dependencies reference nonexistent parent node (u'taggit', u'0004_historicaltag_history_change_reason')
```

**Comment by macro1** [comment]
Ouch. That is unfortunate. I'm surprised this has not come up before. When this app was originally written, migrations were not nearly as common as they are now.

Have you tried using `register()` and passing in `app` to register the history models to an app under your source control?

**Comment by rossmechanic** [comment]
Thinking this is has the same fix as #395 .

**Comment by rossmechanic** [comment]
Passing in `app` will allow the migrations to be moved to a different app
